src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie

```diff
@@ -0,0 +1,88 @@
+
+On May 25th, customers will see an option in the guided installation process to set up their Kubernetes Clusters by including the Pixie Beta alongside the New Relic Infrastructure agent in their Helm chart or Manifest file. 
+
+The New Relic Pixie integration is not capturing all the data, we are capturing metrics and spans that give an overview of how the service is performing, which allows the customer to find problematic areas. Once they have spotted an issue, they might be able to figure it out through data captured in New Relic. However in many cases the issue will need to be debugged in Pixie by diving deeping into the actual requests (headers, content). The headers and content can contain personal information, which stays in the customer's Kubernetes cluster and is not sent to New Relic. In the Pixie UI they can query that information though PXL scripts.
+
+
+What is Pixie auto-telemetry?
+Benefits to New Relic customers (why it matters)
+get observability into Kubernetes services without installing an APM agent 
+run a single command to deploy the new Kubernetes experience
+Benefits to existing Pixie customers
+Integrate Pixie data into New Relic
+Data retention
+Alerting
+Capabilities of the Pixie integration
+get visibility into HTTP services using golden metrics, HTTP transactions, database transactions, distributed tracing and JVM metrics.
+operate, debug and scale Kubernetes cluster by getting insight into how their cluster and services are running. Start from the cluster, go into namespaces, dive deep into deployments and pods, and at each level, see the key metrics and events.
+See anomalous behaviour in their services. Dive deeper by installing an APM 
+
+## Install Auto-Telemetry with Pixie and start instrumenting your Kubernetes clusters
+
+Use our [Guided installation](/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview/) process to install Auto-Telemetry with Pixie. This process installs Pixie on the cluster together with our Kubernetes integration. You don't need to do any further configuration or installation to start using Pixie.
+
+If you want to install Auto-telemetry with Pixie on multiple clusters, you have to run through the Guided install steps for each individual cluster.
+
+**Before you begin**
+
+The guided install requires a Pixie login and Pixie API key.
+
+Currently, Auto-Telemetry with Pixie requires a New Relic account and a separate Pixie account. You can [create the Pixie account](https://work.withpixie.ai/admin/api-keys) before you begin or when needed as part of the install steps. 
```

Comment:
I would add: you will need a Google account to sign up for Pixie.

Reply:
added